Cargo.toml

```diff
@@ -4,7 +4,8 @@ edition = "2021"
 version = "0.1.0"
 
 [dependencies]
-ed25519-dalek = { version = "1.0.1", features = ["serde"] }
+#ed25519-dalek = { version = "1.0.1", features = ["serde"] }
```

Comment:
clean up

Reply:
Commented line removed and change pushed to branch.